### ISSUE TITLE
Problème pour configurer le .env 

### ISSUE THREAD

**Issue by lefarmer01600** [jul-fls/wigpi#1]
Salut, je suis un étudiant de EPSI Lyon et j'aimerais utiliser ton projet pour améliorer notre emploi du temps.

J'ai un problème à la configuration du .env, je ne sais pas quel lien je dois mettre dans AUTH_SERVICE_URL et WIGOR_BASE_URL.

Merci d'avance.

**Comment by jul-fls** [comment]
Salut,
Ce projet étant basé sur du reverse engineering de la plateforme du CAS (portail d'authentification du réseau C&D) ainsi que de la plateforme d'emploi du temps du réseau C&D hébergé sur le site wigorservices.net (d'où le nom wigpi = wigor + api) il est assez compliqué à utiliser / déployer.
Le mieux que je puisse faire pour t'aider, serait que tu m'ajoutes sur Discord afin que l'on puisse se caler un créneau pour que je t'explique comment tout cela s'articule étant donné que ce projet est en réalité lié à d'autres microservices présentés ici : [Organisation Github Wigpi](https://github.com/orgs/wigpi/repositories).
Mon pseudo discord : julfls

**Comment by lefarmer01600** [comment]
Ok, en fait, j'ai essayé de comprendre ton code et je n'arrivais pas à comprendre comment tu obtenais les cookies de la platform CAS, mais je comprends mieux maintenant, je n'avais pas tout le code du projet, je vais essayer de le mettre en place de mon côté et si j'ai un problème, je te contacterai sur discord.

Merci pour ta réponse 

PS : Petit conseille, tu devrais mettre le lien de l'organisation dans ce repository, parce que en faisant des recherches, je suis tombé sur cette page et l'organisation ne ressort pas, ça serait plus simple si d'autre personne d'EPSI veulent essayer d'utiliser ton projet

**Comment by jul-fls** [comment]
Yes, en fait ce repo est à l'origine du projet que je considère un peu comme un repo legacy et l'objectif est de réimplenter toutes les features du code de ce repo dans les autres microservices de l'organisation github (notamment le backend-api-service) mais ceci est comme on dirait WIP (en cours quoi), l'objectif final est de se passer de ce repo et d'utiliser uniquement les microservices de l'orga d'où le fait que ce ne soit pas référencé mais je vais l'ajouter le temps de terminer la migration, bonne idée.

**Comment by lefarmer01600** [comment]
Oui, j'ai vu ça, franchement bravo, j'ai voulu faire un projet comme ça, mais je me suis rendu compte de son ampleur, et j'ai vu qu'il y avait déjà ton projet.

Bon courage en tout cas
